docs/docs/reference/metaprogramming/tasty-reflect.md

```diff
@@ -29,22 +29,19 @@ import scala.quoted._
 inline def natConst(x: => Int): Int = ${natConstImpl('{x})}
 
 def natConstImpl(x: Expr[Int])(using qctx: QuoteContext): Expr[Int] = {
-  import qctx.tasty.{_, given _}
+  import qctx.tasty._
   ...
 }
 ```
 
-### Sealing and Unsealing
+### Extractors
 
-`import qctx.tasty.{_, given _}` will provide an `unseal` extension method on `quoted.Expr`
-and `quoted.Type` which returns a `qctx.tasty.Term` that represents the tree of
-the expression and `qctx.tasty.TypeTree` that represents the tree of the type
-respectively. It will also import all extractors and methods on TASTy Reflect
+`import qctx.tasty._` will provide all extractors and methods on TASTy Reflect
 trees. For example the `Literal(_)` extractor used below.
 
 ```scala
 def natConstImpl(x: Expr[Int])(using qctx: QuoteContext): Expr[Int] = {
-  import qctx.tasty.{_, given _}
+  import qctx.tasty._
   val xTree: Term = x.unseal
   xTree match {
     case Inlined(_, _, Literal(Constant(n: Int))) =>
```

Comment:
Line 64 of this file:

typo: `qctx.tasty.Term.seal[T]`
should be: `qctx.tasty.Term.seal`

Reply:
Fixed